### ISSUE TITLE
formatString returns an array?

### ISSUE THREAD

**Issue by bingqiao** [stefalda/react-localization#46]
Why does it not return a formatted string instead? Or is there something I'm not doing right?
Thanks

**Comment by stefalda** [comment]
Could you show me your code? it should return a string...

**Comment by bingqiao** [comment]
```
import * as Localization from 'react-localization';

const LocalizedStrings = (Localization as any).default;

export const localeStrings = new LocalizedStrings({
    'fr': {
        ok: "OK"
    }
});

export function localize(stringToLocalize: string) {
    return localeStrings.formatString(localeStrings[stringToLocalize]);
}

```

Thanks

**Comment by stefalda** [comment]
You should have something like:

`return localeStrings.formatString(localeStrings[stringToLocalize], "first param", "second param")`

and `localeStrings[stringToLocalize]` should be something like "{0} and {1}".

**Comment by bingqiao** [comment]
Thanks for the comment.

**Comment by stefalda** [comment]
It's a problem with React v16.0.
If the parameter is a string it works ok, if it's JSX it fails...

**Comment by tom025** [comment]
Looks like this was resolved by c5aac1bf40972edac30ef1692edbb8556c9b1d51
